Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Pandaria Winters

## Developer changelog:
- Implemented winters in Pandaria.


# How to test:
Enter the game and check the province climates (it's visible when you hover the mouse over the province), tell your feedback.
